features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,32 @@
+Given("the following user exists:") do |table|
+  table.hashes.each do |table|
+    User.create(table)
+  end
+end
+
+Given("I visit the page") do
+  visit '/'
+end
+
+When("I click on {string}") do |login|
+  click_on(login)
+end
+
+When("I fill in {string} field with {string}") do |email, myEmail|
```

Comment:
please use some other iteration variables, please use snake case instead of camel case